src/current/cockroachcloud/managed-backups-advanced.md

```diff
@@ -239,4 +240,37 @@ For each restore job, the tab will display:
 
 ## Cloud API
 
-{% include cockroachcloud/backups/cloud-api-get-put.md %}
\ No newline at end of file
+{% include cockroachcloud/backups/cloud-api-get-put.md %}
+
+## CockroachDB Cloud Terraform provider
+
+You can use the [CockroachDB {{ site.data.products.cloud }} Terraform provider]({% link cockroachcloud/provision-a-cluster-with-terraform.md %}) to specify managed backup settings in {{ site.data.products.advanced }} clusters.
+
+In your `main.tf` Terraform configuration file, use the `backup_config` attribute on the `cockroach_cluster` resource to modify the settings of managed backups. For example:
+
+{% include_cached copy-clipboard.html %}
+~~~ hcl
+resource "cockroach_cluster" "advanced" {
+  name           = "cockroach-advanced"
+  cloud_provider = "GCP"
+  plan           = "ADVANCED"
+  dedicated = {
+    storage_gib  = 15
+    machine_type = "n2-standard-4"
```

Comment:
We generally don't include machine_type in the terraform examples because it requires a feature flag.  We should replace this with `num_virtual_cpus = 4`

Reply:
I changed this on the Advanced page. I think I grabbed this from the GH repo, my mistake. Thanks for the call out on that.

Reply:
hmm, yeah we should remove those from the examples. I'll file an issue.

Reply:
filed: https://cockroachlabs.atlassian.net/browse/CC-30769